doc/lsp.txt

```diff
@@ -264,6 +264,9 @@ showDiagOnStatusLine	Show a diagnostic message on a status line.
 			By default this is set to false.
 ignoreMissingServer	Do not print a missing language server executable.
 			By default this is set to false.
+showDiagInPopup		Use a popup to display server diagnostics instead of
+			echoing.
+			By default this is set to false.
```

Comment:
By default, is this option set to true or false?

Reply:
Wait, i've forgot to change it back to false, i was testing it

Reply:
Sorry, i've changed it back to false